### ISSUE TITLE
nuget exe v2.8.5 cannot download package running on Mono

### ISSUE THREAD

**Issue by haf** [NuGet/Home#2800]
https://www.nuget.org/packages/FAKE/2.17.9

```
➜  logary git:(master) mono --debug tools/NuGet.exe install FAKE -OutputDirectory tools -Version 2.17.9 -ExcludeVersion
Unable to find version '2.17.9' of package 'FAKE'.
➜  logary git:(master) mono --debug tools/NuGet.exe -version
Unknown command: '-version'
➜  logary git:(master) mono --debug tools/NuGet.exe
NuGet Version: 2.8.50506.491
usage: NuGet <command> [args] [options]
Type 'NuGet help <command>' for help on a specific command.

Available commands:

 config      Gets or sets NuGet config values.

 delete      Deletes a package from the server.

 help (?)    Displays general help information and help information about other commands.

 install     Installs a package using the specified sources. If no sources are specified, all sources defined i
             n the NuGet configuration file are used. If the configuration file specifies no sources, uses the
             default NuGet feed.

 list        Displays a list of packages from a given source. If no sources are specified, all sources defined
             in %AppData%\NuGet\NuGet.config are used. If NuGet.config specifies no sources, uses the default N
             uGet feed.

 pack        Creates a NuGet package based on the specified nuspec or project file.

 push        Pushes a package to the server and publishes it.
             NuGet's default configuration is obtained by loading %AppData%\NuGet\NuGet.config, then loading an
             y nuget.config or .nuget\nuget.config starting from root of drive and ending in current directory.

 restore     Restores NuGet packages.

 setApiKey   Saves an API key for a given server URL. When no URL is provided API key is saved for the NuGet ga
             llery.

 sources     Provides the ability to manage list of sources located in %AppData%\NuGet\NuGet.config

 spec        Generates a nuspec for a new package. If this command is run in the same folder as a project file
             (.csproj, .vbproj, .fsproj), it will create a tokenized nuspec file.

 update      Update packages to latest available versions. This command also updates NuGet.exe itself.

For more information, visit http://docs.nuget.org/docs/reference/command-line-reference
```

Expected outcome; package should be downloaded.

**Comment by rrelyea** [comment]
@haf - Looks like you are trying version 2.8.5 of NuGet.exe
Can you please try a more recent version 3.4.3 or 3.4.4-rc and tell us if that is improved? (http://dist.nuget.org/index.html)

**Comment by zhili1208** [comment]
Verified this and it does not repro this issue on Linux 3.19.0-56-generic, nuget version is 3.4.3.855, mono version is 4.2.1.124.

**Comment by rrelyea** [comment]
closing as it appears fixed in NuGet 3.4.3 and later.
